crates/bevy_scene/src/serde.rs

```diff
@@ -82,30 +90,30 @@ impl<'a> Serialize for EntitySerializer<'a> {
         let mut state = serializer.serialize_struct(ENTITY_STRUCT, 1)?;
         state.serialize_field(
             ENTITY_FIELD_COMPONENTS,
-            &ComponentsSerializer {
-                components: &self.entity.components,
+            &SceneMapSerializer {
+                reflect_array: &self.entity.components,
                 registry: self.registry,
             },
         )?;
         state.end()
     }
 }
 
-pub struct ComponentsSerializer<'a> {
-    pub components: &'a [Box<dyn Reflect>],
+pub struct SceneMapSerializer<'a> {
+    pub reflect_array: &'a [Box<dyn Reflect>],
```

Comment:
Nit: This could also be called `entries` or `values` or something else now that we're not calling this `ReflectArraySerializer`.